assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -110,6 +118,24 @@ export default function usePagePreviewSize(options = {}) {
     availableContainerSpace,
   ]);
 
+  useLayoutEffect(
```

Comment:
Not sure if I'm seeing the need for layout effects here, but guess it doesn't hurt.

I think the order of events will fire like this with this `useLayoutEffect`, but not 100% sure:
```
// -> button clicked 
// -> wait 250ms for button to not be clicked again
// -> fire debounced listener which fires `setWpMenuWidth`
// -> render caused by `wpMenuWidth` update
// -> useLayoutEffect fires before DOM paint calling `setAvailableContainerSpace`
// -> DOM paints here
// -> render caused by `availableContainerSpace` update
// -> final DOM paint
```
I really think `useLayoutEffect` is only useful if you're getting or setting an DOM node's size directly in the effect and need this to be coordinated before a specific paint, but I still don't think this is even needed a lot of time in a lot of animation stuffs